pom.xml

```diff
@@ -165,6 +165,20 @@
                     </execution>
                 </executions>
             </plugin>
+            <!-- Sources jar -->
+            <plugin>
```

Comment:
We already have this in the release profile, do we need it 2 times?

Reply:
I didn't notice, sorry. Why would you want it in a profile though? If I just `mvn clean install` locally, I would want to see sources. Can I remove it from the profile?

Reply:
I think you can remove it from the profile. The release profile is used both for snapshots and production releases

Reply:
Ok as is now it's fine